doc/GettingStarted.md

```diff
@@ -140,7 +200,7 @@ In addition, Vitess requires the software and libraries listed below.
 
     ``` sh
     export MYSQL_FLAVOR=MariaDB
-    or
+    or (mandatory for OSX)
```

Comment:
Let's make this a comment by prefixing `#` (after the indent). Same below on line 215.